source/_integrations/electrasmart.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: Electra Smart
+description: Instructions for how to integrate Electra Air Conditioners within Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.11
```

Comment:
This seems to be incorrect? This is released in the past?

CC @jafar-atili @thecode

Reply:
I had this in mind that we need to fix it before release and forgot. @jafar-atili is on a short vacation, I'll address the comments in a new PR